blog/2023-07-28-july-releases.md

```diff
@@ -0,0 +1,124 @@
+* Plugins are registered using their canonical/fully-qualified name instead of the easily-spoofable entry point name.
+* Subcommand parsing has been refactored to use a greedy parser since `argparse.REMAINDER` has [known issues](https://github.com/python/cpython/issues/61252).
+
+
+### 🔧 What Got Fixed? 🔧
+
+The following bug fixes were implemented in the 23.7.0 version of conda:
+
+* `requests.exceptions.JSONDecodeError` is utilized for ensuring compatibility with different `json` implementations used by requests. This fixes a bug that caused only the first of multiple given source URLs to be tried. This also raises the minimum required requests version to 2.27.0.
+* `__osx` virtual package are no longer exported when `CONDA_OVERRIDE_OSX` is set to an empty string.
+* Erroneous `conda deactivate` behavior that unset pre-existing environment variables that are identical to those set during `conda activate` has been fixed.
+* Third-party subcommands now correctly receive _remaining_ arguments instead of a blanket `sys.argv[2:]`, which broke `conda_cli` testing.
+
+
+### 📄 What's New in Documentation? 📄
+
+* `pre_commands` and `post_commands` plugin hooks.
+* Docstrings for all public modules have been added.
+* API docs are now auto-generated using `sphinx-autoapi`.
+* All manual redirects have been converted into config using `sphinx-reredirects`.
+* The plugins index page has been revised to make it easier to understand how to create a conda plugin.
+* Missing `conda env` CLI docs have been added.
+
+
+### 🌅 What's Marked for [Deprecation](https://github.com/conda-incubator/ceps/blob/main/cep-9.md)? 🌅
+
+The following modules and functions were marked for deprecation:
+
+* [`conda.base.context.context.root_dir`](https://github.com/conda/conda/pull/12701) (use `conda.base.context.context.root_prefix` instead)
+* [`conda.plugins.subcommands.doctor.cli.get_prefix`](https://github.com/conda/conda/pull/12725) (use `conda.base.context.context.target_prefix` instead)
+* [`conda.models.leased_path_entry.LeasedPathEntry`](https://github.com/conda/conda/pull/12735)
+* [`conda.models.enums.LeasedPathType`](https://github.com/conda/conda/pull/12735)
+* [`conda.common.temporary_content_in_file`](https://github.com/conda/conda/pull/12795) (use `tempfile` instead)
+* [`conda.cli.python_api`](https://github.com/conda/conda/pull/12796) (use `conda.testing.conda_cli` fixture instead)
+
+
+* * *
+
+## [Changes in Conda-Build](https://github.com/conda/conda-build/releases/tag/3.26.0)
+
+To update conda-build, run:
+```bash
+conda install -n base conda-build=3.26.0
+```
+
+### ✨ What's New? ✨
+
+In the latest version of conda-build, extra-meta data is logged to make it easier to verify that the right extra-meta data is burned into packages (which also helps to co-relate packages and their build-log). The feature was first introduced in [PR #4303](https://github.com/conda/conda-build/pull/4303) and is now improved via the logging call.
```

Comment:
My brain rebelled against the syntax of "extra-meta data."  Would there be less cognitive dissonance it that is rendered as "`extra-meta` data" instead?